src/Algorithms/BFFPSV18/reach_inhomog.jl

```diff
@@ -226,3 +226,70 @@ function reach_inhomog_BFFPSV18!(F, Xhat0, Φ::MT, NSTEPS, δ::N,
     end
     return F
 end
+
+
+
+
+
+#######################################################################
+
+# Set representation: Generic
+# Matrix operations: Dense
+# Invariant: LazySet
+function reach_inhomog_BFFPSV18!(F, Xhat0, Φ::MT, NSTEPS, δ, X::LazySet, U,
+                                 ST, vars, block_indices,
+                                 row_blocks::AbstractVector{<:RBLKi},
+                                 column_blocks::AbstractVector{<:CBLKj},
+                                 time_shift::N,
+                                 viewval::Val{false}) where {NM, MT<:AbstractMatrix{NM}, N, RBLKi, CBLKj}
```

Comment:
This one is
```suggestion
                                 viewval::Val{true}) where {NM, MT<:AbstractMatrix{NM}, N, RBLKi, CBLKj}
```
because the buffer uses views, see `SMT` below.